subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -523,7 +523,7 @@ void ll_cis_create(uint16_t cis_handle, uint16_t acl_handle)
 	cis->lll.acl_handle = acl_handle;
 
 	/* Create access address */
-	err = util_aa_le32(cis->lll.access_addr);
+	err = util_aa_isr_le32(cis->lll.access_addr, 1U);
```

Comment:
`ll_cis_create()` executes in Thread context (HCI commands from Host). Is it incorrect to use `isr_context` as `true`.

Reply:
Yes you're right. In my workspace I had the code from before your change, where `util_aa_le32` is called in the CIS_IND preparation. When preparing the PR I didn't consider the impact of that change. Thanks.